web/src/rest/publish.js

```diff
@@ -0,0 +1,134 @@
+import axios from 'axios';
+import Vue from 'vue';
+import toggles from '@/featureToggle';
+
+// Ensure contains trailing slash
+const publishApiRoot = process.env.VUE_APP_PUBLISH_API_ROOT.endsWith('/')
+  ? process.env.VUE_APP_PUBLISH_API_ROOT
+  : `${process.env.VUE_APP_PUBLISH_API_ROOT}/`;
+
+function girderize(publishedDandiset) {
+  const { // eslint-disable-next-line camelcase
+    created, modified, dandi_id, version, metadata, name,
+  } = publishedDandiset;
+  return {
+    created,
+    updated: modified,
+    version,
+    name,
+    lowerName: dandi_id,
+    meta: {
+      dandiset: metadata,
+    },
+  };
+}
+
+const client = axios.create({ baseURL: publishApiRoot });
+
+const publishRest = new Vue({
+  data() {
+    return {
+      client,
+      token: null,
+      user: null,
+    };
+  },
+  methods: {
+    // TODO proper OAuth login
+    async logout() {
+      // TODO proper session logout
```

Comment:
By this do you mean server side logout of the session as well?

Reply:
Yes, logout should invalidate the session token. That endpoint doesn't exist yet though.